lib/auth/oidc.go

```diff
@@ -711,7 +711,7 @@ type gsuiteGroup struct {
 	Email string `json:"email"`
 }
 
-// mergeClaims merges b into a.
+// mergeClaims merges b into s.
```

Comment:
I think the search and replace might have gone a bit far with this comment.

Reply:
oops, fixed